core/market/decentralized/tests/test_cyclic_broadcasts.rs

```diff
@@ -107,9 +107,6 @@ async fn test_unsubscribes_cyclic_broadcasts() -> Result<(), anyhow::Error> {
 
     let market3 = network.get_market("Node-3");
 
-    // Create some offers before we instantiate 3rd node.
-    // This way this 3rd node won't get them in first broadcasts, that
-    // are sent immediately, after subscription is made.
```

Comment:
Why is this removed?

Reply:
because this is not true in this test. Node-3 was created already in line 99